tests/pipeline/resources/read-from-component/pipeline.yaml

```diff
@@ -0,0 +1,16 @@
+- type: producer
+  name: producer
+  to:
+    topics:
+      ${output_topic_name}:
+        type: output
+
+- type: streams-app
+  name: consumer1
+
+- type: streams-app
+  name: consumer2
+  from:
+    topics: {}
```

Comment:
What if I omit `topics`?

Reply:
`topics` can't be omitted, this would give a validation error

Reply:
But we could make it an empty map by default?

Reply:
yes

Reply:
I've changed it